_examples/010-examples.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Quick Start  
 layout: toc-guide-page
-lprev: 020-tutorial_qs.html 
-lnext: 015-Prerequisite.html
+lprev: 010-examples.html 
+lnext: 010-examples.html
```

Comment:
Its doesn't seem to make sense that this page's prev and next links point to itself.